### DISCUSSION TITLE
Update Custom clients

### DISCUSSION THREAD

**Discussion by Nawaki89** [rustdesk/rustdesk-server-pro#318] [Ideas]
I'm not really knowledgeable about programming but I had an idea.

I've been sorting out how I should update our clients once there is a newer build to push out.  Of course you could use an RMM software to do this.   But in our company such software is missing in our infrastructure for now.  

But since the clients are communicating with the server encrypted with its API for user account etc.

Should it be possible that once you generate a new build of a custom client that the clients connected to the server could sense this and display there is an update available and download it from the server?  One could code the custom clients with a unique number or tag so it knows which custom client it should check for if there is a newer version.    This would mean the server side need to keep its generated clients(and on the server side it could display a build can be newly generated to a higher version) in its system but for a business side storage at least for me is no issue gladly allocate more system power.

Just a thought I had, might be a massive thing to do.

**Comment by ConGp20** [comment]
Update-functionality also for the normal clients would be great
